suite-native/module-accounts-management/src/components/AccountDetailHeader.tsx

```diff
@@ -65,7 +63,9 @@ export const AccountDetailHeader = ({
     const account = useSelector((state: AccountsRootState) =>
         selectAccountByKey(state, accountKey),
     );
-    const { shallDisplayBaseCurrency } = useDisplayBaseCurrency(account?.symbol);
+    const { shallDisplayBaseCurrency: isNotBaseCurrencyAccount } = useDisplayBaseCurrency(
```

Comment:
Previous naming was better than what you changed it to IMO. It's not perfect, but more understandable of what it implies. Better naming would be something like `isBaseCurrencySameAsAccountCurrency`
```suggestion
    const { shallDisplayBaseCurrency } = useDisplayBaseCurrency(
```

Reply:
This is kind of unfortunate naming IMO. If you see the JSX part, you can see that BaseCurrency component is rendered always so rendering also Crypto componente only in case when `shoulDisplayBasecurrency` is true is kind of weird. I tried to clarify that with the rename, bud I agree that it ended up even more confusing. I am removing this rename in [0fe8ef6](https://github.com/trezor/trezor-suite/pull/22614/commits/0fe8ef64c46c3fb468c1321900c3a9077a3a858a).

Best solution would be to refactor this hook API completely, but that is something I would like to avoid in this PR.